src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -223,21 +218,21 @@ const getIsSupportedOnRainbowWeb = (chainId: ChainId) => {
   }
 };
 
-const UniqueTokenExpandedState = ({ asset: passedAsset, external }: UniqueTokenExpandedStateProps) => {
+const UniqueTokenExpandedState = ({ asset, external }: UniqueTokenExpandedStateProps) => {
```

Comment:
if expensive could memo

Reply:
This is memoized at the bottom of the file.